.gitignore

```diff
@@ -14,7 +14,6 @@ MANIFEST
 *.egg
 *.egg-info
 dist
-build
```

Comment:
I think you still want to ignore this, don't you?  As opposed to `BUILD`.  (Case-insensitive file systems might have trouble there.)  `build` is used by distutils, IIRC.